napari/utils/events/_evented.py

```diff
@@ -0,0 +1,9 @@
+from typing import Protocol, runtime_checkable
+
+from .event import EmitterGroup
```

Comment:
can you rename this file `_protocols.py` (we can put your other Mutable in here later too)

Reply:
Yup, good point, done!